packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx

```diff
@@ -389,8 +399,8 @@ export function VerifyCertificatePage() {
       .join(', ')
   }
 
-  const getRegistarData = (data: any) => {
-    const history = data.history.find(
```

Comment:
Wonder why no errors appear when using "any" here, we should look into enabling that. We have gateway types so there is no reason to do this.